src/Tizen.NUI.Wearable/src/public/CircularSlider.cs

```diff
@@ -0,0 +1,674 @@
+        {
+            var instance = (CircularSlider)bindable;
+            if (newValue != null)
+            {
+                instance.privateIsEnabled = (bool)newValue;
+            }
+        },
+        defaultValueCreator: (bindable) =>
+        {
+            var instance = (CircularSlider)bindable;
+            return instance.privateIsEnabled;
+        });
+
+        private static readonly string TrackVisualName = "Track";
+        private static readonly string ProgressVisualName = "Progress";
+        private static readonly string ThumbVisualName = "Thumb";
+        private ArcVisual trackVisual;
+        private ArcVisual progressVisual;
+        private ArcVisual thumbVisual;
+
+        private float maxValue = 100;
+        private float minValue = 0;
+        private float currentValue = 0;
+        private Size thumbSize;
+        private bool isEnabled = true;
+
+        private EventHandler<ValueChangedArgs> valueChangedHandler;
+
+        float sliderPadding = 6.0f;
+
+        private Animation sweepAngleAnimation;
+        private Animation thumbAnimation;
+
+        /// <summary>
+        /// Get style of progress.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public new CircularSliderStyle Style => ViewStyle as CircularSliderStyle;
+
+        /// <summary>
+        /// Value Changed event data.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public class ValueChangedArgs : EventArgs
+        {
+            /// <summary>
+            /// Curren value
+            /// </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            public float CurrentValue;
+        }
+
+        #endregion Fields
+
+
+        #region Constructors
+
+        static CircularSlider() { }
+        /// <summary>
+        /// The constructor of CircularSlider.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public CircularSlider() : base(new CircularSliderStyle())
+        {
+            Initialize();
+        }
+
+        /// <summary>
+        /// The constructor of the CircularSlider class with specific style.
+        /// </summary>
+        /// <param name="progressStyle">The style object to initialize the CircularSlider.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public CircularSlider(CircularSliderStyle progressStyle) : base(progressStyle)
+        {
+            Initialize();
+        }
+
+        #endregion Constructors
+
+
+        #region Properties
+
+        /// <summary>
+        /// The value changed event handler.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public event EventHandler<ValueChangedArgs> ValueChangedEvent
```

Comment:
If you don't need another work except for add/remove handlers, Event Invoke code can be simplified.
```
public event EventHandler<ValueChangedArgs> ValueChangedEvent
{
   ...
   ...
}
...
...
if (valueChangedHandler != null)
{
    ValueChangedArgs args = new ValueChangedArgs();
    args.CurrentValue = currentValue;
    valueChangedHandler(this, args);
}
```
==>
```
public event EventHandler<ValueChangedArgs> ValueChangedEvent;
...
...
ValueChangedEvent?.Invoke(this, new ValueChangedArgs() { CurrentValue = currentValue });
```